src/main/java/sirius/db/mixing/properties/EnumProperty.java

```diff
@@ -75,6 +75,28 @@ public Object transformValue(Value value) {
         return Value.of(defaultValue).asEnum((Class<Enum>) field.getType());
     }
 
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    @Override
+    protected Object transformValueFromImport(Value value) {
+        if (value.isFilled()) {
+            // If a value is present, we check if any constant name or its translation matches...
+            String stringValue = value.asString().trim();
+            for (Enum enumValue : ((Class<Enum>) field.getType()).getEnumConstants()) {
+                // Check for a match of the constant...
+                if (Strings.equalIgnoreCase(enumValue.name(), stringValue)) {
+                    return enumValue;
+                }
+
+                // Check of a match of the translation...
+                if (Strings.equalIgnoreCase(enumValue.toString(), stringValue)) {
+                    return enumValue;
+                }
+            }
```

Comment:
Sollte man das nicht in 2 Schleifen aufteilen und zuerst nach dem Enum-Namen auflösen, und danach nach dem toString-Wert? Es könnte ja Konflikte geben, wenn eine Konstante gleich heißt wie die Übersetzung einer anderen. Ist aber zugegebenermaßen etwas konstruiert.

Reply:
Funktioniert die Übersetzung an der Stelle überhaupt korrekt? Ich meine, das wird ja oft in Jobs aufgerufen, und ohne UserContext o.ä.?

Reply:
Doch da läuft der user context desjenigen der den job startet. Also gehen nicht alle sprachen, aber die aktuelle des users

Reply:
Wegen den 2 schleifen hatte ich auch schon überlegt aber halte ich für hochstgradig unwahrscheinlich. Das würde uns ja auch beim debuggen etc abfucken